<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HDF Guide</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="leftSplitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="GroupBrowserWidget" name="explorer">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
      <widget class="QWidget" name="mainwidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="DatasetSearchWidget" name="searchLineEdit">
          <property name="toolTip">
           <string/>
          </property>
          <property name="toolTipDuration">
           <number>0</number>
          </property>
          <property name="statusTip">
           <string/>
          </property>
          <property name="whatsThis">
           <string/>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="placeholderText">
           <string/>
          </property>
          <property name="clearButtonEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSplitter" name="centralSplitter">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="opaqueResize">
           <bool>true</bool>
          </property>
          <widget class="TerminalWidget" name="console"/>
          <widget class="QStackedWidget" name="stackedWidget">
           <widget class="AnalysisViewWidget" name="editor"/>
           <widget class="QTableView" name="data">
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="preview" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QSplitter" name="middleSplitter">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="opaqueResize">
           <bool>true</bool>
          </property>
          <widget class="DatasetPreviewWidget" name="summary">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="acceptDrops">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="AttributeTableWidget" name="attributes">
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutRole</enum>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AnalysisViewWidget</class>
   <extends>QTextEdit</extends>
   <header>AnalysisViewWidget.h</header>
  </customwidget>
  <customwidget>
   <class>TerminalWidget</class>
   <extends>QPlainTextEdit</extends>
   <header>TerminalWidget.h</header>
  </customwidget>
  <customwidget>
   <class>GroupBrowserWidget</class>
   <extends>QTreeWidget</extends>
   <header>GroupBrowserWidget.h</header>
  </customwidget>
  <customwidget>
   <class>DatasetPreviewWidget</class>
   <extends>QTextEdit</extends>
   <header>DatasetPreviewWidget.h</header>
  </customwidget>
  <customwidget>
   <class>AttributeTableWidget</class>
   <extends>QTableWidget</extends>
   <header>AttributeTableWidget.h</header>
  </customwidget>
  <customwidget>
   <class>DatasetSearchWidget</class>
   <extends>QLineEdit</extends>
   <header>DatasetSearchWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionAbout</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onAbout()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onAbout()</slot>
 </slots>
</ui>
